src/components/List/RowContent.js

```diff
@@ -162,6 +172,64 @@ const MobileContainer = styled.div`
   `};
 `;
 
+const MultilinePrimaryText = styled(PrimaryText).attrs({
+  className: "text--primary"
+})`
+  overflow: hidden;
+  position: relative;
+  line-height: 1.2em;
```

Comment:
we have defined line height for text in `theme/typography.js`

Reply:
okay, will add those.